com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Recursive-Rendering.md

```diff
@@ -29,7 +29,7 @@ You can also do this for Shader Graph master nodes:
 1. In the Project window, double-click on the Shader to open it in Shader Graph.
 2. In the **Graph Settings** tab of the **Graph Inspector**, go to **Surface Options** and enable **Recursive Rendering (Preview)**.
 
-It is best practice to use recursive rendering in situations that require multi-bounced reflection and refraction, for example car headlights. Although recursive rendering also produces ray-traced reflections in more simple scenarios, like for a mirror or a puddle, it is best practice to use [ray-traced reflections](Ray-Traced-Reflections.md) here for performance reasons.
+It is best practice to use recursive rendering in situations that require multi-bounced reflection and refraction, for example car headlights. Although recursive rendering also produces recursive rendering in more simple scenarios, like for a mirror or a puddle, it is best practice to use [ray-traced reflections](Ray-Traced-Reflections.md) here for performance reasons.
```

Comment:
Let's improve the phrasing here: 

It is best practice to use recursive rendering in situations that require multi-bounced reflection and refraction, for example, car headlights. You can use recursive rendering in simple scenarios, like a mirror or a puddle, but for best performance, use [ray-traced reflections](Ray-Traced-Reflections.md).

Reply:
not sure when i changed this, probably a find and replace